### ISSUE TITLE
Remove vendor prefix from commands in AT Driver's "settings" module

### ISSUE THREAD

**Issue by jugglinmike** [Prime-Access-Consulting/nvda-at-automation#48]
[In March, the AT Driver proposal removed vendor prefixing from its "settings" module.](https://github.com/w3c/at-driver/pull/65)

**Comment by jscholes** [comment]
@jugglinmike Am I understanding correctly that the "`nvda:`" prefix should be entirely removed from all settings-related commands?  I.e.:

* `nvda:settings.getSettings` -> `settings.getSettings`;
* `nvda:settings.getSupportedSettings` -> ``settings.getSupportedSettings`; and
* `nvda:settings.setSettings` -> `settings.setSettings`?

**Comment by jugglinmike** [comment]
@jscholes Yup, that's right!

**Comment by jugglinmike** [comment]
This has been resolved by gh-57.
